### ISSUE TITLE
Expose url generation function from react-imgix to support custom url generation

### ISSUE THREAD

**Issue by cainlevy** [imgix/react-imgix#131]
**type:** feature request

**react-imgix version:** `v1.0`

**Code**

```jsx
import {toURL} from 'react-imgix';

<video poster={toURL(...)}>...</video>
```

**Expected behavior:**

react-imgix exports a function that returns a Imgix URL with as many of the react-imgix options and defaults as make sense. (no auto-sizing)

**Actual behavior:**

react-imgix only exports a component

**Workaround:**

Do not use `<video poster />`. Instead render an image with a click handler that swaps out for the video and begins playback.

**Comment by frederickfogerty** [comment]
Hey @cainlevy we've decided that this is worth implementing. We can't commit to a timeline for this yet, but I'll keep this issue updated 👍

**Comment by frederickfogerty** [comment]
Released in `8.4.0` 🎉
